client/blocks/reader-share/style.scss

```diff
@@ -198,6 +208,5 @@
 .reader-share__site-selector .site__domain {
 	&::after {
 		@include long-content-fade();
-		border-radius: 50%;
```

Comment:
This definitely makes it look a bit better when the colors are mismatched, but now that they are matched this change isn't observable.

> This may not be 100% necessary, before I realized the --color-surface above was wrong it was glaring that the fade was not the correct shape and I removed this first fixing its shape. After fixing --color-surface above, the change of border radius really has no effect, but it seems unnecessary now so we can probably remove.